docs/sources/panels-visualizations/query-transform-data/transform-data/index.md

```diff
@@ -272,6 +272,9 @@ Use this transformation to change the field type of the specified field.
   - **Time -** attempts to parse the values as time
     - Will show an option to specify a DateFormat as input by a string like yyyy-mm-dd or DD MM YYYY hh:mm:ss
   - **Boolean -** will make the values booleans
+  - **Enum -** will make the values enums
+    - Will show a table to manage the enums
+  - **Other -** attempts to parse the values as JSON
```

Comment:
```suggestion
  - **Enum** - will make the values enums
    - Will show a table to manage the enums
  - **Other** - attempts to parse the values as JSON
```
Just made a quick change to the new syntax, and again, this should be updated in the `content.ts` file, the script mentioned at the top of the file run, then the markdown should reflect the change.